pkg/reconciler/v1alpha1/eventlistener/eventlistener_test.go

```diff
@@ -995,12 +1027,29 @@ func TestReconcile(t *testing.T) {
 			Services:       []*corev1.Service{elService},
 			ConfigMaps:     []*corev1.ConfigMap{loggingConfigMap},
 		},
+	}, {
+		name:   "eventlistener with SetSecurityContext false",
```

Comment:
I added this unit test. It is now possible to set a config for each test, ensuring that flags are honored properly.